pkg/query_expr/es_expr_test.go

```diff
@@ -0,0 +1,97 @@
+package queryexpr_test
+
+import (
+	"testing"
+
+	queryexpr "github.com/goto/compass/pkg/query_expr"
+)
+
+func TestESExpr_String(t *testing.T) {
+	tests := []struct {
+		expr queryexpr.SQLExpr
+		want string
+	}{
+		{
+			expr: queryexpr.SQLExpr("test"),
+			want: "test",
+		},
+		{
+			expr: queryexpr.SQLExpr("bool_identifier == !(findLast([1, 2, 3, 4], # > 2) == 4)"),
+			want: "bool_identifier == !(findLast([1, 2, 3, 4], # > 2) == 4)",
+		},
+	}
+	for i, tt := range tests {
+		t.Run("test-case-"+string(rune(i)), func(t *testing.T) {
+			if got := tt.expr.String(); got != tt.want {
+				t.Errorf("String() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestESExpr_ToQuery(t *testing.T) {
```

Comment:
unit test seems to be small in number, while the implementation is quite long. does it cover the critical requirements?

Reply:
Yes, currently only cover the critical requirements

Reply:
I'm not sure if we have checked all, can't really tell as query checking is very difficult. For now, I just assume you have ensured its safety. Resolving this.

Reply:
cc: @haveiss